### ISSUE TITLE
Tweaking past sponsors listing

### ISSUE THREAD

**Issue by peterjc** [OBF/homepage#22]
We have logos of past sponsors here:

https://www.open-bio.org/events/bosc/sponsors/

None of the logos are clickable links - should they be?

The current display always using the same width has left some of the logos are comparatively small (e.g. GigaScience), and others larger (e.g. the squares for F1000 or PeerJ). I would like to make the landscape logos a little wider, and the square ones a little smaller.

**Comment by nlharris** [comment]
Yes, I had noticed that the past sponsor logos are not linked. I feel that's ok, as that's a benefit for our current sponsors and not something we should be responsible for in perpetuity, especially as the links may change in time.
I agree with your suggestion about adjusting logo sizes.
